reportportal_client/helpers.pyi

```diff
@@ -1,20 +1,18 @@
 from logging import Logger
-from typing import Dict, List, Text
+from typing import Dict, List, Text, Union
 
 logger: Logger
 
+def generate_uuid() -> Text: ...
 
-def generate_uuid() -> str: ...
-
-
-def convert_string(value: str) -> str: ...
-
+def convert_string(value: Text) -> Text: ...
 
 def dict_to_payload(value: Dict) -> List[Dict]: ...
 
-
-def gen_attributes(rp_attributes: List) -> List[Dict]: ...
+def gen_attributes(rp_attributes: List[Dict]) -> List[Dict]: ...
 
 def get_launch_sys_attrs() -> Dict[Text]: ...
 
 def get_package_version(package_name: Text) -> Text: ...
+
+def verify_value_length(attributes: Union[List[Dict], None]) -> List[Dict]: ...
```

Comment:
It looks like the function returns Optional[List[Dict]], as attributes can be None